tiatoolbox/models/classification/patch_predictor.py

```diff
@@ -325,7 +318,6 @@ def _predict_engine(
             return_labels (bool): Whether to return labels.
             return_coordinates (bool): Whether to return patch coordinates.
             on_gpu (bool): whether to run model on the GPU.
-
         Returns:
             output (ndarray): Model predictions of the input dataset
 
```

Comment:
Add an example.

Reply:
This is an internal function and is not expected to be used by user. I'm not sure it needs example.